### PR TITLE
Build fixes for file-split-ftp sample

### PR DESCRIPTION
**Pull request by deki**
The file-split-ftp sample didn't run after clone, here are some minor fixes to get it working.

### CODE REVIEW

File: applications/file-split-ftp/pom.xml [hunk 1]
```diff
@@ -4,7 +4,6 @@
   <groupId>org.springframework.integration.samples</groupId>
   <artifactId>file-split-ftp</artifactId>
   <version>6.0.0</version>
-  <packaging>pom</packaging>
```
Comment (artembilan) [inline applications/file-split-ftp/pom.xml:7]: The project is based on a Gradle, so if there are some fixes needed they have to be made in the `build.gradle`.
It looks like this packaging is wrong whenever we add Spring Boot Maven plugin (or its parent).

**Review by artembilan — CHANGES_REQUESTED** [review]
It would be great to know what is really the problem with the mentioned application before start doing some fixes.

Thanks

File: applications/file-split-ftp/pom.xml (hunk 1 above)
Comment (deki) [reply applications/file-split-ftp/pom.xml:7]: I don't see a build.gradle file in the file-split-ftp directory.

**Comment by artembilan** [comment]
Here is a fix: https://github.com/spring-projects/spring-integration-samples/commit/3326699377d979c884c2935cb4856dec496750f1.

Turns out the real problem was with Java Mail dependencies where we have to pull a proper Jakarta Activation one.
As you see all the POMs have been regenerated as well.

File: applications/file-split-ftp/pom.xml (hunk 1 above)
Comment (artembilan) [reply applications/file-split-ftp/pom.xml:7]: There is only a single top-level one for the whole project.

**Comment by deki** [comment]
The blocker was `<packaging>pom</packaging>` because that skips compilation. Wasn't aware that the pom.xml is generated but it's fixed with your commit as well.
Thank you!

**Pull request closed without merging**